docs/content/docs/dev/table/sqlClient.md

```diff
@@ -169,559 +169,394 @@ Mode "embedded" (default) submits Flink jobs from the local machine.
+print the causes why the execution fails. For detailed exception stack, please set the `sql-client.verbose` true.
 
-**A literal value with explicit type:** Explicitly declare the parameter with `type` and `value` properties for type-safety.
+### Execute SQL Files
 
-```yaml
-- type: DECIMAL
-  value: 11111111111111111
-```
+When start up the SQL Client with -f option, the SQL Client will split the content of the specified file into multiple statements and submit them line by line.
+The SQL Client will print all the messages as the message printed in the interactive mode.
+An example of such a file is presented below.
 
-The table below illustrates the supported Java parameter types and the corresponding SQL type strings.
-
-| Java type               |  SQL type         |
-| :---------------------- | :---------------- |
-| `java.math.BigDecimal`  | `DECIMAL`         |
-| `java.lang.Boolean`     | `BOOLEAN`         |
-| `java.lang.Byte`        | `TINYINT`         |
-| `java.lang.Double`      | `DOUBLE`          |
-| `java.lang.Float`       | `REAL`, `FLOAT`   |
-| `java.lang.Integer`     | `INTEGER`, `INT`  |
-| `java.lang.Long`        | `BIGINT`          |
-| `java.lang.Short`       | `SMALLINT`        |
-| `java.lang.String`      | `VARCHAR`         |
-
-More types (e.g., `TIMESTAMP` or `ARRAY`), primitive types, and `null` are not supported yet.
-
-**A (nested) class instance:** Besides literal values, you can also create (nested) class instances for constructor parameters by specifying the `class` and `constructor` properties.
-This process can be recursively performed until all the constructor parameters are represented with literal values.
-
-```yaml
-- class: foo.bar.paramClass
-  constructor:
-    - StarryName
-    - class: java.lang.Integer
-      constructor:
-        - class: java.lang.String
-          constructor:
-            - type: VARCHAR
-              value: 3
+```sql
+CREATE TEMPORARY TABLE users (
+  user_id BIGINT,
+  user_name STRING,
+  user_level STRING,
+  region STRING,
+  PRIMARY KEY (user_id) NOT ENFORCED
+) WITH (
+  'connector' = 'upsert-kafka',
+  'topic' = 'users',
+  'properties.bootstrap.servers' = '...',
+  'key.format' = 'csv',
+  'value.format' = 'avro'
+);
+
+-- set sync mode
+SET table.dml-sync=true;
+
+-- set the job name
+SET pipeline.name=SqlJob;
+
+-- set the queue that the job submit to
+SET yarn.application.queue=root;
+
+-- set the job parallism
+SET parallism.default=100;
+
+-- restore from the specific savepoint path
+SET execution.savepoint.path=/tmp/flink-savepoints/savepoint-cca7bc-bb1e257f0dab;
+
+INSERT INTO pageviews_enriched
+SELECT *
+FROM pageviews AS p
+LEFT JOIN users FOR SYSTEM_TIME AS OF p.proctime AS u
+ON p.user_id = u.user_id;
 ```
+This configuration:
 
-{{< top >}}
+- defines a temporal table source `users` that reads from a CSV file,
+- set the properties, e.g job name,
+- set the savepoint path,
+- submit a sql job that load the savepoint from the specified savepoint path.
```

Comment:
Would be better to separate them into different sections, becuase they are frequently asked questions.